enricher/fabric8/src/main/java/io/fabric8/maven/enricher/fabric8/SpringBootHealthCheckEnricher.java

```diff
@@ -59,7 +62,11 @@ protected Probe getLivenessProbe() {
     private Probe discoverSpringBootHealthCheck(int initialDelay) {
         try {
             if (MavenUtil.hasAllClasses(this.getProject(), REQUIRED_CLASSES)) {
-                Properties properties = SpringBootUtil.getSpringBootApplicationProperties(this.getProject());
+                String strActiveProfiles = getContext().getConfig().getConfig("spring-boot","activeProfiles");
+
+                Properties properties = SpringBootUtil.getApplicationProperties(getContext().getProject(),
+                        SpringBootUtil.getActiveProfiles(strActiveProfiles));
+                
```

Comment:
I would make this a single method in the utility class, so that not two calls are required to the same class. should be easy to do.